### PR TITLE
Translate ruby 2.4.0-preview1 released (zh)

### PR DESCRIPTION
**Pull request by hlcfan**
@JuanitoFatas please 👀.
I'm not good at Traditional Chinese tho, please point out if any improper places.

### CODE REVIEW

File: zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+layout: news_post
+title: "Ruby 2.4.0-preview1 发布"
+author: "naruse"
+translator: "Alex S"
+date: 2016-06-20 14:00:00 +0000
```
Comment (stomar) [inline zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: wrong date

File: zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+## [字符串支持 Unicode 大小写及其他类型](https://bugs.ruby-lang.org/issues/10085)
+
+`String/Symbol#upcase/downcase/swapcase/capitalize(!)` 现在可以做 Unicode 的的大小写及其他类型，而不仅仅是 ASCII 类型转换。
+
+## 性能提升
+
+Ruby 2.4 还包括以下性能提升，和一些语法改进：
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
```
Comment (stomar) [inline zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: duplicate heading

File: zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+### [Thread#report_on_exception 和 Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+Ruby 忽视线程中的异常，除非另一个线程显式地执行直至结束。
+通过设置 `report_on_exception = true`，你会注意到如果一个线程终止了因为未处理的异常。
+
+请向我们反馈 `report_on_exception` 的默认值应该是什么和垃圾回收报告。
+
+### [线程死锁检查现在会显示线程的栈和依赖](https://bugs.ruby-lang.org/issues/8214)
+
+Try and enjoy programming with Ruby 2.4.0-preview1, and [send us feedback](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)!
```
Comment (stomar) [inline zh_cn/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: untranslated

File: zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+---
+layout: news_post
+title: "Ruby 2.4.0-preview1 發布"
+author: "naruse"
+translator: "Alex S"
+date: 2016-06-20 14:00:00 +0000
```
Comment (stomar) [inline zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: wrong time stamp

File: zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+lang: zh_cn
```
Comment (stomar) [inline zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: zh_tw

File: zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+## [字符串支持 Unicode 大小寫及其他類型](https://bugs.ruby-lang.org/issues/10085)
+
+`String/Symbol#upcase/downcase/swapcase/capitalize(!)` 現在可以做 Unicode 的的大小寫及其他類型，而不僅僅是 ASCII 類型轉換。
+
+## 性能提升
+
+Ruby 2.4 還包括以下性能提升，和一些語法改進：
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
```
Comment (stomar) [inline zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: duplicate

File: zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+### [Thread#report_on_exception 和 Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+Ruby 忽視線程中的異常，除非另一個線程顯式地執行直至結束。
+通過設置 `report_on_exception = true`，你會注意到如果一個線程終止了因為未處理的異常。
+
+請向我們反饋 `report_on_exception` 的默認值應該是什麼和垃圾回收報告。
+
+### [線程死鎖檢查現在會顯示線程的棧和依賴](https://bugs.ruby-lang.org/issues/8214)
+
+Try and enjoy programming with Ruby 2.4.0-preview1, and [send us feedback](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)!
```
Comment (stomar) [inline zh_tw/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: untranslated

**Comment by hlcfan** [comment]
Updated, thank you for review! @stomar

**Comment by JuanitoFatas** [comment]
@stomar Thanks for review.

@hlcfan Merged manually in a7bd0d7.

It is very nice that you also sent a PR for zh_tw, but if that is directly machine-translated post from zh_cn, then please leave to me next time, thanks! :)

**Comment by hlcfan** [comment]
@JuanitoFatas Awesome, thanks.

**Pull request closed without merging**